flow/connectors/postgres/cdc.go

```diff
@@ -48,6 +53,7 @@ type PostgresCDCConfig struct {
 	SrcTableIDNameMapping  map[uint32]string
 	TableNameMapping       map[string]model.NameAndExclude
 	RelationMessageMapping model.RelationMessageMapping
+	CatalogConnection      *pgxpool.Pool
```

Comment:
can we call it `CatalogPool` (i'm seeing it as pool in `PostgresCDCSource` but connection here)

Reply:
can we store `flowJobName` here as well? It will make it easy to access everywhere downstream.